bftengine/src/bcstatetransfer/InMemoryDataStore.cpp

```diff
@@ -88,8 +88,6 @@ std::vector<std::pair<BlockId, Digest>> InMemoryDataStore::getPrunedBlocksDigest
 
 void InMemoryDataStore::setCheckpointDesc(uint64_t checkpoint, const CheckpointDesc& desc) {
   ConcordAssert(checkpoint == desc.checkpointNum);
-  ConcordAssert(descMap.count(checkpoint) == 0);
```

Comment:
its better to keep the assert, and find a way from the caller to avoid it from failing:

- new flag - less preferable (flag set to false by default, and if true , jump the assert)
- delete the descripto (zero it?) before the call - better way.

Reply:
I went with new flag as it was more convenient compared to bringing in whole new API.